### PR TITLE
SORTING

### PR DESCRIPTION
**Pull request by salalo**
#10

### CODE REVIEW

File: lib/flat_search_web/live/page_live.ex [hunk 1]
```diff
@@ -18,10 +24,14 @@ defmodule FlatSearchWeb.PageLive do
       |> Filters.change_filter(params)
       |> Map.put(:action, :insert)
 
-    params = Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end)
+      ic_params = Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end)
```
Comment (boksadawid) [inline lib/flat_search_web/live/page_live.ex]: ic_params?

Comment (salalo) [reply lib/flat_search_web/live/page_live.ex]: text there is formatted to be case insensitive, naming is hard :P ic -> ci?

Comment (boksadawid) [reply lib/flat_search_web/live/page_live.ex]: is there any need to keep this and params separately?

Comment (salalo) [reply lib/flat_search_web/live/page_live.ex]: Later on you can see use of both original params and the ic_params.

Comment (salalo) [reply lib/flat_search_web/live/page_live.ex]: I'll look into it and try to get rid of ic

Comment (salalo) [reply lib/flat_search_web/live/page_live.ex]: That's my another idea but not sure if that makes a huge difference :p

File: lib/flat_search_web/live/page_live.ex [hunk 2]
```diff
@@ -18,10 +24,16 @@ defmodule FlatSearchWeb.PageLive do
...
-
     {:noreply,
-     assign(socket, changeset: changeset, flats: Flats.get_flats_by(params), filters: params)}
+     assign(socket,
+       changeset: changeset,
+       flats:
+         Flats.get_flats_by(
+           Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end),
+           params["order_by"]
+         ),
```
Comment (boksadawid) [inline lib/flat_search_web/live/page_live.ex]: maybe move this above the tuple and assign to variable `flats`

Comment (salalo) [reply lib/flat_search_web/live/page_live.ex:34]: I can't  override the original param because I still use it

Comment (salalo) [reply lib/flat_search_web/live/page_live.ex]: I did it this way not to allocate memory for the list for only one use

**Review by boksadawid — APPROVED** [review]

**Pull request merged**